### PR TITLE
tooltips updated accroding to manual (synced using G1ANT.Manual.Builder)

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**

### CODE REVIEW

File: G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md
Comment (Vegebutcher) [inline G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md:8]: I changed it in the description to:
This command clicks a desktop application UI element specified by WPath structure.

**Review by Vegebutcher — CHANGES_REQUESTED** [review]

**Review by przegwar — APPROVED** [review]

File: G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md
Comment (WiktoriaPrusik) [reply G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md:8]: But you changed it after I did this sync so you should approve.

Comment (WiktoriaPrusik) [reply G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md:8]: @Vegebutcher I merged develop to this branch and synced again so please approve :)

**Review by Vegebutcher — APPROVED** [review]

**Pull request merged**
